src/test/java/com/cag/cagbackendapi/service/producer/UserServiceTest.kt

```diff
@@ -0,0 +1,74 @@
+package com.cag.cagbackendapi.service.producer
+
+import com.cag.cagbackendapi.model.UserModel
+import com.cag.cagbackendapi.service.user.UserService
+import com.nhaarman.mockito_kotlin.*
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.mockito.InjectMocks
+import org.mockito.MockitoAnnotations
+import org.mockito.junit.jupiter.MockitoExtension
+import org.slf4j.Logger
+
+@ExtendWith(MockitoExtension::class)
+class UserServiceTest {
+
+    private var logger: Logger = mock()
+
+    @InjectMocks
+    private lateinit var userService: UserService
+
+    @BeforeEach
+    fun setUp() {
```

Comment:
For Kotlin fun is function right, I looked it up and wanted to be sure I understood it right. Does it need to be said every time a function is used in Kotlin or just when a new function is made.

Reply:
"fun" is just for declaring functions in kotlin!